ush/python/pygfs/task/analysis.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from netCDF4 import Dataset
+from typing import List, Dict, Any
+
+from pygw.yaml_file import YAMLFile
+from pygw.file_utils import FileHandler
+from pygw.logger import logit
+from pygw.task import Task
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class Analysis(Task):
+    """Parent class for GDAS tasks
+
+    The Analysis class is the parent class for all
+    Global Data Assimilation System (GDAS) tasks
+    directly related to peforming an analysis
+    """
+
+    def __init__(self, config: Dict[str, Any]) -> None:
+        super().__init__(config)
+        self.config.ntiles = 6
```

Comment:
I just realized, we have a mechanism for it.
```suggestion
        super().__init__(config, ntiles=6)
```